docs/after/qet.md

```diff
@@ -1,38 +1,28 @@
----
-id: qet
-title: QET
----
-
-The QET is an English Language test set by the Centre for English Language Communication. It must be taken by students who do not possess the necessary English Language qualifications when they enter the university.
-
-
-Date: 14 July 2020 <br/> Time: 9.30 am to 11.15 am
-
-We strongly recommend you to take QET this July and not to postpone it as it is something that you would want to get over with quickly!
-
-:::note
-You need at least a band 2 result to stay in [Residential Colleges](residential-colleges.md)
-:::
-
-For latest information, find out more about QET's [general information, preparation](http://www.nus.edu.sg/celc/programmes/qet.html) and [detailed information, timetables and exemptions](http://www.nus.edu.sg/registrar/academic-activities/registration/academic-related-matters/qet)
-
-### Registration for Briefing Sessions
-
-:::caution
-It is compulsory for students to attend at least one briefing session.
-Students will **NOT** be allowed to sit for the QET if they are absent from the briefing session even if they have registered for the QET.
-:::
-
-Therefore, please plan your arrival according to these [briefing sessions](http://www.nus.edu.sg/celc/programmes/files/Briefing%20registration%20notice.pdf), which usually take place the day before the QET itself!
-
-### Registration for QET
-
-Remember to register online via [Edurec](https://myedurec.nus.edu.sg/) from 19 June to 28 June and follow the instructions [here](http://www.nus.edu.sg/registrar/docs/info/academic-activities/registration/user-guide-for-qet-det-online-registration-checking-of-test-results.pdf). 
-
-### What to prepare?
-Usually, Perhimpunan Indonesia NUS will provide QET tutoring session to brief and prepare you for the exam! Wait for the announcement!
-
-You must bring laptops, since this will be an online based test.
-
-### Frequently Asked Questions
-1. What if I am exempted from taking QET? What should I do? <br/> _Freshmen who have met the exemption criteria can overlook this step entirely. There is no need to report your fulfillment of the criteria to CELC, so just relax._
+---
+id: qet
+title: QET
+---
+
+The QET is an English Language test set by the Centre for English Language Communication. It must be taken by students who do not possess the necessary English Language qualifications when they enter the university.
+
+
+
```

Comment:
why so many newlines?

Reply:
I think this applies to some other parts of the docs as well, can you perhaps just scan through the docs once to ensure consistent formatting -- sorry for the troubles

Reply:
Whoops sorry forgot to check the formatting again